contracts/mainnet/connectors/permit_erc20/main.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity ^0.7.6;
+pragma experimental ABIEncoderV2;
+
+import {TokenInterface, MemoryInterface} from "../../common/interfaces.sol";
+import {Stores} from "../../common/stores.sol";
+import {ERC20_functions, ERC20_dai_functions} from "./interface.sol";
+//import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+
+/**
+ * @title permit_erc20.
+ * @dev Adding permit functionality to ERC_20.
+ */
+
+
+contract permit_erc20 {
+    address dai_address = 0x6B175474E89094C44Da98b954EedeAC495271d0F; // dai has a different implementation for permit
```

Comment:
We use camel casing not `dashes`.
```suggestion
    address private immutable dai_address = 0x6B175474E89094C44Da98b954EedeAC495271d0F; // dai has a different implementation for permit
```

Reply:
Changed it, please have a look and let me know if any edit is needed.

Reply:
https://github.com/harshit-wq/dsa-connectors/blob/e7386b993a6052709120dec2b65f43418fb96e62/contracts/mainnet/connectors/permit_erc20/main.sol#L17